README.md

```diff
@@ -31,24 +31,26 @@ When a major issue in "beta 3" is found on SOAK, we will follow the steps above
 Only pre-releases that are SOAKed for at least two weeks should be used as stable releases. After the soaking period was successful, follow these steps:
 
 1. fetch latest repo state: `git fetch`
-1. checkout the pre-release tag you want to make a stable release: e.g. `git checkout v1.1.0-rc.3`
-1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`
-1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to your newly created `stable-*` branch (remember to also update revision)
+1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to `master` (remember to also update revision)
 1. apply and push tags
    1. "base" semver based tag: e.g. `git tag v1.1.0 && git push origin v1.1.0`
    1. "consumable" stable tag(s) for supported k8s versions: e.g. `git tag stable-1.16-1.1.0 && git push origin stable-1.16-1.1.0`
+1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`. that way future updates have an easy target.
 1. head to github and update release information for that release: check [releases page](https://github.com/mesosphere/kubeaddons-kommander/releases) for up-to-date example. _Make sure to edit the "base" release_.
-1. add that new stable branch to `mergebot-config.json` on `master` and set its version to the next patch release (usually `.1`)
+1. add that new `stable-*` branch to `mergebot-config.json` on `master` and set its version to the next patch release (usually `.1`)
+1. in order to allow backports to that newely reated minor version, make sure that the charts minor version also is bumped. 
+1. to make it easy for fellow colleagues, create a new directory (e.g. `1.2.0` for people to work in) 
 
 ### Dealing with previously released stable versions
 
 Sometimes we might need to push a fix for an older version, in these cases we need to use the `stable-*` branch for these versions. E.g. in order to be able to push "Kommander 1.0.1" after `master` already is in a WIP "1.1.x" state, we have `stable-1.0.x` branch to release `v1.0.1` tag.
 
-#### Tag/release previously released versions
+#### Tag/release previously released versions (patch releases)
 
 1. fetch latest repo state: `git fetch`
 1. checkout respective stable branch: e.g. `git checkout stable-1.0.x`
 1. apply tags:
    1. apply and push "base" semver based tag: e.g. `git tag v1.0.1 && git push origin v1.0.1`
    1. apply and push "consumable" stable tag(s) for each supported k8s version: e.g. `git tag stable-1.16-1.0.1 && git push origin stable-1.16-1.0.1`
+1. merge that `stable-*` branch back to `master` in order to keep all information on master
```

Comment:
```suggestion
1. merge that `stable-*` branch back to `master` in order to keep all information on master **NOTE: `master` should NEVER be merged into the `stable-*` branch.**
```
Adding note here that we should never merge master back into any stable branches to avoid getting newer revisions on the stable branch which would be used instead of the older patch revisions. We should also discuss and document here -- should the merge of stable back into master happen via direct commit on the master branch? Or do we want to PR this out -- in which case, the PR will be blocked on the "out of date" warning and requires admin privileges to override the block and merge the PR anyways.

Reply:
my 2c: i think opening PRs is good practice generally because 1) you get more eyes on it 2) it gets tested. we might run into a situation one day where the stable-x.y branch _cannot_ be merged back into master bc of a breaking change. That would be good to know and share on a PR and thus we'll know when a divergence occurs.

That said, if we are going with PRs we have to make doubly sure that we don't instinctively click "update with latest master" as Github will tell us to or we revert those master-merges in the same PR if we do it by accident. I myself will be more diligent about it going forward.

Reply:
> should the merge of stable back into master happen via direct commit on the master branch? Or do we want to PR this out

there should never be a conflict and it should only be a fast-forward merge, therefore just merging stable into master by commit would be fine for me. if theres a conflict, something went wrong anyway.

@samvantran made a good point, Im worried about this "update with latest master" button too, also theres "delete branch after merge" 🙈